app/models/twitter_user.rb

```diff
@@ -120,32 +120,32 @@ def build_relations
       ]
       if without_friends
         candidates = candidates.slice(2, candidates.size - 2)
-        _statuses, _search_results, _, _mentions, _favorites = client.fetch_parallelly(candidates)
+        _statuses, _search_results, _, _mentions, _favorites = client._fetch_parallelly(candidates)
         _friends = _followers = []
       else
-        _friends, _followers, _statuses, _search_results, _, _mentions, _favorites = client.fetch_parallelly(candidates)
+        _friends, _followers, _statuses, _search_results, _, _mentions, _favorites = client._fetch_parallelly(candidates)
```

Comment:
__[rubocop]__
* Line is too long. [121/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)